lib/src/main/java/io/ably/lib/GlobalConstants.java

```diff
@@ -0,0 +1,11 @@
+package io.ably.lib;
+
+public class GlobalConstants {
+
+    public static final String LIB_VERSION = "0.8.2";
```

Comment:
Perhaps update https://github.com/ably/ably-java/blob/master/README.md as it states that there are version references but does not include a mention of this new one now